### ISSUE TITLE
Greenshot Release|x64 fails to build on VS 16.7 Preview 3

### ISSUE THREAD

**Issue by swaroop-sridhar** [dotnet/project-system#6280]
Documenting a failure reported by the SDK/CTI team.

**Visual Studio Version**:
VS 16.7 Preview 3

**Summary**:

Greenshot Release|x64 fails to build because:
* The Greenshot project still seems to be building for AnyCPU
* Dependent Projects are skipped.

**Steps to Reproduce**:

* git clonehttps://github.com/swaroop-sridhar/greenshot -b develop
* Open src\Greenshot.sln in VS 2019
* Select the configuration:  Platform=X64 Configuration=Release.
* Build the Greenshot.csproj

**Expected Behavior**:

Build succeeds

**Actual Behavior**:

Build fails

```
>------ Skipped Build: Project: Greenshot.Gfx ------
2>------ Skipped Build: Project: Greenshot.Core ------
3>------ Skipped Build: Project: Greenshot.Addons ------
4>------ Skipped Build: Project: Greenshot.Addon.ExternalCommand ------
5>------ Skipped Build: Project: Greenshot.Addon.Office ------
6>------ Skipped Build: Project: Greenshot.Addon.LegacyEditor ------
7>------ Skipped Build: Project: Greenshot.Addon.InternetExplorer ------
8>------ Build started: Project: Greenshot, Configuration: Release Any CPU ------
...

MC1000  Unknown build error, 'Could not find a part   of the path   'C:\Users\v-damu\Downloads\greenshot-develop\greenshot-develop\src\Greenshot.Addons\bin\x64\Release\netcoreapp3.0\Greenshot.Addons.dll'.   Line 13 Position 10.'
```

**Notes**:

* Building from command line using SDK 5.0.100-preview.6.20301.7 works OK
* Building using VS 16.6.2 works OK
* Building using VS 16.7 Preview 3  Release|AnyCPU works OK
* Building using VS 16.7 Preview 3 Release|X64 after it is once correctly built by an earlier version works OK.
* Building using VS 16.7 Preview 3 Release|X64  works OK if 16.6.2 is installed on the machine.

**Comment by jjmew** [comment]
Looking at the solution configuration we realized that it is mis-configured. For Release x64 it is configured to build "Any CPU". 

![image](https://user-images.githubusercontent.com/14163814/84547714-f144cf80-acb8-11ea-9fab-0fcbc20ca2f1.png)

**Comment by swaroop-sridhar** [comment]
OK Adding the following change: https://github.com/swaroop-sridhar/greenshot/pull/1 seems to fix the build.

It is still surprising that the old project works on 16.6.2 but not on 16,7*. So, maybe this is undefined behavior, or a bug in older builds.

**Comment by dandanmu** [comment]
Fixed on the environment : Version 16.7 peview 3.0 b30212.156 + .net 5.0 SDK:5.0.100-preview.6.20314.3

**Comment by drewnoakes** [comment]
It's unclear what action we could take here to improve debugging this misconfigured solution. Closing this for now.
